src/api/stories.ts

```diff
@@ -21,3 +21,20 @@ export const listUserStories = async <T = Stories.ListUserStoriesResponse>(
       },
     })
   )
+
+/**
+ * 获取用户的日记视频地址
+ * @param id 视频日记id
+ */
+export const getStoryVideo = async <T = Stories.StoryVideoResponse>(
```

Comment:
The filename and method name should be consistent with api url.
Please move it to `media-meta.ts`, and rename the method to `interactive`